cli/src/cli_util.rs

```diff
@@ -2293,7 +2293,10 @@ See https://jj-vcs.github.io/jj/latest/working-copy/#stale-working-copy \
             .commits(repo.store())
             .try_collect()?;
 
-        if !root_conflict_commits.is_empty() {
+        let should_show_resolve_conflicts_hint =
```

Comment:
nit: I would move this to the function start since we don't have to compute the revset if disabled. This function is actually `show_hint_on_how_to_resolve_conflicts()`.